src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -37,6 +37,11 @@
 
         String p;
 
+        p = config.getInitParameter("userToken");
+        if (p != null && p.length() > 0) {
+            this.projectToken = p;
```

Comment:
this.projectToken is how it's stored in the code.

The section below it ensures that projectToken takes priority, because if it's configured, it will replace userToken.